bootstrap/kubeadm/api/v1alpha3/kubeadmbootstrapconfig_types.go

```diff
@@ -68,6 +68,11 @@ type KubeadmConfigSpec struct {
 	// Format specifies the output format of the bootstrap data
 	// +optional
 	Format Format `json:"format,omitempty"`
+
+	// Verbosity is the number for the kubeadm log level verbosity.
+	// It overrides the `--v` flag in kubeadm commands.
+	// +optional
+	Verbosity int `json:"verbosity,omitempty"`
```

Comment:
Would it make sense to make this a `*int` instead? That would allow us to better differentiate unset vs explictly set to 0.

Reply:
Is 0 a valid value for klog? If it is, then +!

Reply:
I was mainly thinking it would help with the somewhat awkward logic around testing if it is set to `"0"` to determine not to specify the the `--v` flag. If 0 isn't a valid value (or there is a specific max) we can validate those through kubebuilder annotations/openapi validation

Reply:
yeah good point

Reply:
ah yeah that's true, although that would move the checks for nil somewhere else, no strong opinions here, either way seems fine to me